code/modules/mob/living/carbon/inventory.dm

```diff
@@ -29,6 +29,9 @@
 	else if(I == r_hand)
 		r_hand = null
 
+	if(I.pulledby)
+		I.pulledby.stop_pulling()

```

Comment:
these stop_pulling() here and in handle_item_insertion() and put_in_l_hand() are here so we don't need to check that our pulled thing's loc is still a turf in living/Move().